src/startupscriptgenerator/src/hugo/config.go

```diff
@@ -0,0 +1,11 @@
+// --------------------------------------------------------------------------------------------
```

Comment:
Is there much different between the startup script generator we're adding for Go compared to Hugo, and between Go/Hugo and existing platforms?

Reply:
Currently there are no differences. But as we begin to develop more we will would need separate startups.